src/wechaty_plugin_contrib/contrib/chat_history_plugin/plugin.py

```diff
@@ -2,15 +2,14 @@
 import os
 from typing import Optional, List
 from dataclasses import dataclass, field
-from wechaty_puppet import MessageType  # type: ignore
-from wechaty import Message, get_logger  # type: ignore
+from wechaty_puppet import MessageType, FileBox  # type: ignore
+from wechaty import Wechaty, Message, get_logger  # type: ignore
```

Comment:
Have you already remove the annotation: `# type: ignore` ?

Reply:
Yes, in the new commit, all `# type: ignore` have been removed, the `pyproject.toml` works well for mypy check.